### ISSUE TITLE
Questions. General questions and documentation

### ISSUE THREAD

**Issue by rodrigolzd** [Ylianst/MeshCentral#3012]
I should just point out how awesome Meshcentral is, but I cant, it is just an incredible set of tools. 

Perhaps a thank you is better because this has help me stay afloat during hard times this past few months while dealing with the aftermath of my family being hit with covid-19 and losing my father to it.  
With Meshcentral I was able to keep supporting some of my customers and family without having to go directly to their location and exposing people to this terrible disease.

So, Thank you for all your work!

Some of my questions are pretty simple and at times I wonder if i should really post them here on github, where I think the questions should be more towards development of the software rather than usage and other aspects. I fear I might just pollute this project with simple usage questions that could be better suited for the reddit forum perhaps. I say this because I think the actual core team is pretty small and I see how much is being done already to the point where I think the issue and question I might ask will only take away valuable time.  So, where should I post them? 

Regarding the documentation, I keep wondering if there is a chance the install guide and other documentation could be editable? Perhaps in a google docs or even a wiki format.  Right now the install guide is at Version 0.1.0 from November 28, 2020. From that time until now mesh has had quite a few updates and new functions, I'm sure some of us here bugging you for features and support could also add some info to the documentation at some point. This could also help translating the documentation as well.

Once again, Thank your all for this incredible set of tools!

**Comment by D4V3M0NK** [comment]
I agree. I offered to assist in this effort some time back and then got hit with a ton of work and feel awful that I wasn't able to come good on my offer. I do think though that having wiki or CMS based documentation would be hugely beneficial and take some of the load off of Ylian, Bryan and others. This has been suggested before and I'm sure others would be keen to assist.

Personally I think that here within GitHub would be a good start - it might not be the final resting place for the documentation, but I believe it already has a section where documentation can be edited, editors would have to log into github so that all changes can be tracked. IMHO it serves a simple start to just seeing how the documentation *could* be managed on a longer term basis without committing too much to setting up a more elaborate system to do so (if that was even required). It could even be classed as not being the official documentation until the team sees how the community works with changes etc and then could potentially make a decision on whether or not it's something they can see continuing.

**Comment by silversword411** [comment]
Dive in, I did the first major work, just edit the `/docs/` files.
Fix and improve
